### PR TITLE
HTTP Methods implemented

### PR DESCRIPTION
**Pull request by daniel33gomez**

### CODE REVIEW

**Review by Sam-sta — CHANGES_REQUESTED** [review]
Fix lint errors

**Review by Motopshyco — COMMENTED** [review]
Fix the indentation errors

**Review by rubiperucci — CHANGES_REQUESTED** [review]
please fix lint inconsistencies

**Review by itcr00w — CHANGES_REQUESTED** [review]
review eslint problems

**Review by Sam-sta — APPROVED** [review]

**Review by Motopshyco — APPROVED** [review]

**Review by zkpain — APPROVED** [review]

File: src/main/server.js [hunk 1]
```diff
@@ -1,6 +1,51 @@
...
+
+    res.send(result);
+});
+
+app.get('/scenario', async (req, res) => {
+    const scenario = await db.collection('MSM-Scenario').doc('ExampleId01').get();
+    res.send(scenario.data());
+});
+
+app.listen(3000, function() {
```
Comment (alexeindev) [inline src/main/server.js:49]: Maybe for next iterations, we shouldn't hardcode the port. The rest seem great 👍🏻

**Review by alexeindev — APPROVED** [review]

**Review by itcr00w — APPROVED** [review]

**Review by rubiperucci — APPROVED** [review]

**Pull request merged**